### DISCUSSION TITLE
Question: how to install runtime on linux ?

### DISCUSSION THREAD

**Discussion by boxerab** [microsoft/onnxruntime#6489] [General] [answered]
I have successfully built the runtime on Ubuntu: how do I install into `/usr/local/lib`
so that another application can link to the library ?  Also, is it possible to generate a pkg-config
`.pc` file to manage dependencies ?

**Comment by snnn** [comment, marked as answer]
Please download onnxruntime-linux-x64-1.6.0.tgz
from https://github.com/microsoft/onnxruntime/releases/tag/v1.6.0, then unzip the file and manually copy the files to /usr or /usr/local, either works. Then please run `ldconfig . ` in the lib directory which contains libonnxruntime.so.

We don't provide pkg-config files, but you can write one. onnxruntime doesn't have extra dependencies other than the standard C/C++ libs. You don't need to do extra things for this.

**Reply by boxerab** [reply]
Thanks, @snnn. I am already building from source. I will write a patch to do a standard *nix install and generate pkg-config files.

**Reply by opringle** [reply]
This was really helpful. Docs only provide installation instructions using dotnet. That led me down the rabbit hole of building from source when you can just unzip this release. https://onnxruntime.ai/docs/install/#python-installs
